bridge/discord/webhook.go

```diff
@@ -82,10 +82,8 @@ func (b *Bdiscord) webhookSend(msg *config.Message, channelID string) (*discordg
 				ContentType: "",
 				Reader:      bytes.NewReader(*fi.Data),
 			}
-			content := ""
-			if msg.Text == "" {
-				content = fi.Comment
-			}
+			content := fi.Comment
```

Comment:
```suggestion
			content := fi.Comment
			if msg.Text != "" {
				content = msg.Text
			}
```

Reply:
can you test with this change, so we only add the comment when there is empty text